### PR TITLE
Add HealthCheck

### PR DESCRIPTION
**Pull request by rafal-gorecki**

### CODE REVIEW

**Review by delihus — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -25,12 +25,33 @@ RUN mkdir -p ~/ros2_ws/src && \
...
     rm -rf /var/lib/apt/lists/*
 
+# Create health check package
+RUN cd src/ && \
+    source /opt/ros/$ROS_DISTRO/setup.bash && \
+    ros2 pkg create healthcheck_pkg --build-type ament_cmake --dependencies rclcpp std_msgs && \
+    sed -i '/find_package(std_msgs REQUIRED)/a \
+            find_package(sensor_msgs REQUIRED)\n \
+            add_executable(healthcheck_node src/healthcheck.cpp)\n \
+            ament_target_dependencies(healthcheck_node rclcpp std_msgs sensor_msgs)\n \
```
Comment (delihus) [inline Dockerfile]:
```suggestion
            ament_target_dependencies(healthcheck_node rclcpp sensor_msgs)\n \
```

File: Dockerfile [hunk 2]
```diff
@@ -25,12 +25,33 @@ RUN mkdir -p ~/ros2_ws/src && \
     bash ./src/OrbbecSDK_ROS2/orbbec_camera/scripts/install_udev_rules.sh && \
     rm -rf /var/lib/apt/lists/*
 
+# Create health check package
+RUN cd src/ && \
+    source /opt/ros/$ROS_DISTRO/setup.bash && \
+    ros2 pkg create healthcheck_pkg --build-type ament_cmake --dependencies rclcpp std_msgs && \
+    sed -i '/find_package(std_msgs REQUIRED)/a \
```
Comment (delihus) [inline Dockerfile]: std_msgs is not necessary

**Review by delihus — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 3]
```diff
@@ -25,12 +25,33 @@ RUN mkdir -p ~/ros2_ws/src && \
     bash ./src/OrbbecSDK_ROS2/orbbec_camera/scripts/install_udev_rules.sh && \
     rm -rf /var/lib/apt/lists/*
 
+# Create health check package
+RUN cd src/ && \
+    source /opt/ros/$ROS_DISTRO/setup.bash && \
+    ros2 pkg create healthcheck_pkg --build-type ament_cmake --dependencies rclcpp std_msgs && \
```
Comment (delihus) [inline Dockerfile]:
```suggestion
    ros2 pkg create healthcheck_pkg --build-type ament_cmake --dependencies rclcpp sensor_msgs && \
```

**Review by delihus — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -28,11 +28,10 @@ RUN mkdir -p ~/ros2_ws/src && \
 # Create health check package
 RUN cd src/ && \
     source /opt/ros/$ROS_DISTRO/setup.bash && \
-    ros2 pkg create healthcheck_pkg --build-type ament_cmake --dependencies rclcpp std_msgs && \
-    sed -i '/find_package(std_msgs REQUIRED)/a \
-            find_package(sensor_msgs REQUIRED)\n \
+    ros2 pkg create healthcheck_pkg --build-type ament_cmake --dependencies rclcpp sensor_msgs && \
+    sed -i '/find_package(sensor_msgs REQUIRED)/a \
```
Comment (delihus) [inline Dockerfile:32]: there should be `find_package(rclcpp REQUIRED)` i think

**Review by delihus — CHANGES_REQUESTED** [review]

**Review by delihus — APPROVED** [review]

File: Dockerfile [hunk 5]
```diff
@@ -25,12 +25,32 @@ RUN mkdir -p ~/ros2_ws/src && \
...
+            add_executable(healthcheck_node src/healthcheck.cpp)\n \
+            ament_target_dependencies(healthcheck_node rclcpp sensor_msgs)\n \
+            install(TARGETS healthcheck_node DESTINATION lib/${PROJECT_NAME})' \
+            /ros2_ws/src/healthcheck_pkg/CMakeLists.txt
+
+COPY ./healthcheck.cpp /ros2_ws/src/healthcheck_pkg/src/
+
 # Build
 RUN source /opt/ros/$ROS_DISTRO/setup.bash && \
     colcon build --event-handlers  console_direct+  --cmake-args  -DCMAKE_BUILD_TYPE=Release
```
Comment (rafal-gorecki) [inline Dockerfile]:
```suggestion
    colcon build --cmake-args  -DCMAKE_BUILD_TYPE=Release
```

**Pull request merged**